ocaml/quicktest/quicktest_args.ml

```diff
@@ -10,6 +10,19 @@ let username = ref ""
 let password = ref ""
 
 let host = ref ""
+let localhost_uuid =
```

Comment:
```
let localhost_uuid = Inventory.lookup "INSTALLATION_UUID"
```

Reply:
i tried that, but it doesn't work

at the time I put it down to the fact that the `quicktest` binary is actually a client to xapi, rather than part of the xapi daemon

Reply:
if there is a cleaner way to get the localhost uuid however, i would much rather do that

Reply:
The Inventory module is part of xcp-inventory. Would it be possible to use that? If not, I think the solution is fine.

https://github.com/xapi-project/xcp-inventory

The implementation is a single file. So it could be also vendored.

Reply:
I think adding it to the libraries in dune's executable definition would work, see `ocaml/util/dune`

Reply:
`Xapi_inventory` is already a dependency of `quicktestbin` (presumably via `xapi_internal`)

the reason it wasn't working was because you have to set `inventory_filename`, but that wasn't happening early enough.

Interestingly setting it at the top of main (`quicktest.ml`) doesn't seem to work, so i have set it when we assign `localhost_uuid`